### PR TITLE
Add custom ObjectUtils

### PR DESCRIPTION
**Pull request by And1sS**

### CODE REVIEW

File: src/main/java/org/prebid/server/util/ObjectUtils.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package org.prebid.server.util;
+
+import java.util.Arrays;
+import java.util.Objects;
+import java.util.function.Function;
+import java.util.function.Supplier;
+import java.util.stream.Stream;
+
+public final class ObjectUtils {
```
Comment (rpanchyk) [inline src/main/java/org/prebid/server/util/ObjectUtils.java:9]: Please rename to `ObjectUtil` to be in one style with other util classes.

**Review by rpanchyk — CHANGES_REQUESTED** [review]

File: src/main/java/org/prebid/server/util/ObjectUtils.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        return source != null ? getter.apply(source) : null;
+    }
+
+    public static <T> T defaultIfNull(T object, T defaultValue) {
+        return object != null ? object : defaultValue;
+    }
+
+    public static <T> boolean anyNotNull(T... objects) {
+        return Arrays.stream(objects).anyMatch(Objects::nonNull);
+    }
```
Comment (rpanchyk) [inline src/main/java/org/prebid/server/util/ObjectUtils.java:33]: Redundant method, `org.apache.commons.lang3.ObjectUtils` already has them.

File: src/main/java/org/prebid/server/util/ObjectUtil.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+import java.util.Objects;
+import java.util.function.Function;
+import java.util.function.Supplier;
+import java.util.stream.Stream;
+
+public final class ObjectUtil {
+
+    private ObjectUtil() {
+    }
```
Comment (snahornyi) [inline src/main/java/org/prebid/server/util/ObjectUtil.java:11]: Let's use approach from 'Application' class. Add `@SuppressWarnings("checkstyle:hideutilityclassconstructor")` and remove constructor

Comment (rpanchyk) [reply src/main/java/org/prebid/server/util/ObjectUtil.java:11]: i'd rather to keep it simple and stay with private constructor.

**Review by rpanchyk — APPROVED** [review]

**Pull request merged**